backend/editor/graph_db.py

```diff
@@ -0,0 +1,18 @@
+from neo4j import GraphDatabase         # Interface with Neo4J
+from . import settings                  # Neo4J settings
+
+txn = None
```

Comment:
Ok, now the next step (because we want to support concurrency) is to make it compatible with concurrency (in an async context).

For this, you will need a [context variable](https://docs.python.org/3/library/contextvars.html)

It's better to encapsulate "getting the transaction" in a function. So that in entries.py you have:
`transaction().run(....)`
(note that since you will use a function, you can now import the function directly: `from graph_bd import transaction()`)
This function should return current value of global context variable, or raise if it's not set (or set to None)